### ISSUE TITLE
Image skipping when switching

### ISSUE THREAD

**Issue by Fawaytish** [poppeman/Pictus#76]
When I'm fast switching between images with pressed right/left arrow, I notice that Pictus is skipping some images, probably because CPU is not processing images fast enough. My suggestion to fix it is somehow force Pictus to show every image. (wait for CPU to show the image and then switch to next image)

**Comment by poppeman** [comment]
If you are using ALT+arrow keys (which by default is bound to "Next/Previous few images"), then that is intended behavior. It instantly jumps five images in either direction, not attempting to load anything in between.

**Comment by Fawaytish** [comment]
No, I'm using just arrow keys without ALT. In a large folder of images I hold down right/left arrow key to fast switching between all images. But Pictus is skipping some images, so I have to go slowly with arrow keys to view every image from the folder. I'm only experiencing this problem on a PCs with slow CPUs.
